### ISSUE TITLE
508-defect-3 [COGNITION]: An empty value SHOULD be clearly communicated

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#13550]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

When there isn't a value for a labeled item, it **should** be communicated to the user. 

On this screen the **Status** field has no value, and may be confusing, especially for screen reader users, who may think there's a mistake in the page rendering.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a screen reader user, I want to confidently read the page content and know when a field is empty.

## Environment

* Operating System: all
* Browser: all
* Screenreading device: any
* Server destination: staging

## Steps to Recreate

1. Enter https://staging.va.gov/manage-va-debt/your-debt/ and please use Greg Anderson (user 1).
2. Verify that the Status label does not pair with a value as shown in the enclosed screenshot

## Possible Fixes (optional)

Designers would pair with Peggy Gannon to determine how to communicate that no status value is available, even if the answer is "N/A".

## WCAG or Vendor Guidance (optional)

> The information architecture of the site and design SHOULD be intuitive, allowing the user to easily find, navigate, read, and interact with the content. [Understandability/Cognition](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/resources/WCAG-Checklist.md#part-4-understandabilitycognition)

## Screenshots or Trace Logs

![Screen Shot 2020-09-14 at 5 53 14 PM](https://user-images.githubusercontent.com/57469/93142476-1714b280-f6b4-11ea-91ea-4be38708547b.png)

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

**Comment by bradl3yC** [comment]
Apparently this data was removed from their API response - we are meeting with them today and I will ask them to add it back in

**Comment by bradl3yC** [comment]
@jenstrickland this should be in staging with the next deploy ~30 minutes or so - please validate at your leisure

**Comment by jenstrickland** [comment]
Validated
